docs/en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -406,6 +406,31 @@ the best way to think about it is that the object where the relationship will be
 Product => Categories, the `Product` should contain the `many_many`, because it is much
 more likely that the user will select Categories for a Product than vice-versa.
 
+## Cascading deletions
+
+Relationships between objects can cause cascading deletions, if necessary, through configuration of the
+`cascade_deletes` config on the parent class.
+
+```php
+class ParentObject extends DataObject {
+    private static $has_one = [
+        'Child' => ChildObject::class,
+    ];
+    private static $cascade_deletes = [
+        'Child',
+    ];
+}
+class ChildObject extends DataObject {
+}
+```
+
+In this example, when the Parent object is deleted, the Child specified by the has_one relation will also
+be deleted.
```

Comment:
Does this also apply to `many_many`, `belongs_many_many`, `has_many` and `many_many through` as well?
It seems to according to the docblock in `DataObject`
Perhaps add a note about those types of relationships here to be consistent.